### ISSUE TITLE
Not Using CUDA ?

### ISSUE THREAD

**Issue by vagi8** [mesutpiskin/id-card-detector#29]
The camera version runs better in windows than linux.(although the FPS is better in linux than windows, i believe it is because you are using parallel programming) 

**Linux** 
Ubuntu 16.04 Bionic
Nvidia GTX1060 16GB, 
8GB RAM 
CUDA and Cudnn

**Windows 10**
8GB RAM
No CUDA

Is it not using CUDA or what ?

**Need help making it faster**.

**Comment by mesutpiskin** [comment]
Unfortunately, we would be glad if you contribute the project.

**Comment by vagi8** [comment]
Happy to do so !

**Comment by vagi8** [comment]
Hey ! I ll be needing the data to be retrain the model.

**Comment by mesutpiskin** [comment]
You are right, but there is private information in the dataset. Shall we create a public dataset project? Sounds great!
